core/src/data_sources/qdrant.rs

```diff
@@ -152,3 +160,92 @@ impl QdrantClients {
         }
     }
 }
+
+#[derive(Clone)]
+pub struct DustQdrantClient {
+    client: Arc<QdrantClient>,
+    pub cluster: QdrantCluster,
+}
+
+impl DustQdrantClient {
+    // In v1 implementations, we'll be able:
+    // - we'll be able to retrieve the shared collection name from the data source config.
+    // - we'll be able to add a condition on the PointsSelector to match the
+    //   data_source.internal_id multi-tenancy filter.
+
+    pub async fn create_data_source(&self, data_source: &DataSource) -> Result<()> {
+        // TODO: v0 implementation
```

Comment:
Just to make sure I follow:
- v0, it creates a collection
- v1 it does not do anything?

Reply:
Yes this will be the plan for create_data_source indeed :+1: